Golem/Jobs.cs

```diff
@@ -182,4 +147,69 @@ public Task<List<IJob>> List()
         }
         return null;
     }
+
+    public Task<List<Job>> UpdateJobs(List<ActivityState> activityStates)
+    {
+        return Task.FromResult(activityStates
```

Comment:
It doesn't make sense to keep this method async and wrap the result with a `Task` here.

Reply:
The reason is that this is a blocking call. An equivalent async code would be something like:

```
public async Task<List<Job>> UpdateJobs(List<ActivityState> activityStates)
    {
        var jobTasks = activityStates
            .Select(updateJob);

        var jobs = await Task.WhenAll(jobTasks);
                
        return jobs
            .Where(j => j!=null)
            .Cast<Job>()
            .ToList();        
    }
```